acceptance-tests/tests/helpers/auth.js

```diff
@@ -0,0 +1,48 @@
+const yaml = require('js-yaml');
+const rimraf = require('rimraf');
+const { existsSync, readFileSync, writeFileSync } = require('fs');
+const cmd = require('./cmd');
+const { CONFIG_FILE_NAME } = require('../../lib/constants');
+
+let PARSED_CONFIG_YAML;
+
+async function initializeAuth() {
```

Comment:
This initializes, parses, and then removes the cli config file so we can use the values to easily write new config files later in the tests.